src/System.Management.Automation/security/SecuritySupport.cs

```diff
@@ -595,7 +596,7 @@ internal static void CheckIfFileExists(string filePath)
         /// <returns>True on success, false otherwise.</returns>
         internal static bool CertIsGoodForSigning(X509Certificate2 c)
         {
-            if (!CertHasPrivatekey(c))
+            if (!c.HasPrivateKey)
```

Comment:
It looks like CertHasPrivateKey() is only used here, so it can be removed as dead code.

Reply:
I already removed the method.